roles/glance/tasks/logging.yml

```diff
@@ -3,7 +3,8 @@
   logrotate: name=glance path=/var/log/glance/*.log
   args:
     options:
-      - daily
+      - weekly
```

Comment:
We have. I think this was done prior to that decision. Let me update it and I'll make sure that they are all compliant in the future. Sorry about that.